### PR TITLE
Add extract lyrics test cases

### PR DESCRIPTION
**Pull request by MacyChan**

### CODE REVIEW

File: tests/test_extract_lyrics.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+# Case 2 - empty dataframe
+def test_empty_case():
+    try:
+        token = get_token()
+        arr_edge1 = ["", "Taylor Swift"]
+        with pytest.raises(ValueError):
+            pl.get_lyrics(token, arr_edge1[0], arr_edge1[1])
+        assert pl.get_lyrics(token, arr_edge1[0], arr_edge1[1]) == None, "Edge case exception handling failed, should return None"    
+    except:
+        assert True
```
Comment (artanzand) [inline tests/test_extract_lyrics.py:36]: Do we need the try/except for the last couple of functions?

**Review by artanzand — COMMENTED** [review]
Looks very good. Well done!

File: tests/test_extract_lyrics.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+def get_token():
+    current_path = os.getcwd()
+    with open(current_path + "/tests/data/credentials.json") as f:
+        login = json.load(f)
+    return login["token"]
+
+
+# Case 1 - happy case
+def test_happy_case():
```
Comment (artanzand) [inline tests/test_extract_lyrics.py:19]: Recommending on bringing function comments inside:
```suggestion
def test_happy_case():
"""Test a happy case"""
```

Comment (artanzand) [reply tests/test_extract_lyrics.py:19]: same recommendation for other functions

**Review by artanzand — COMMENTED** [review]
Requested adding short function description

File: tests/test_extract_lyrics.py (hunk 2 above)
Comment (MacyChan) [reply tests/test_extract_lyrics.py:7]: I used try-except to catch my error message, to make sure it throws the correct error.
Since it is a happy case, it should not throw error, so not useful in this function
[Tiff's test](https://github.com/ttimbers/convertempPy/blob/master/tests/test_convertempPy.py) doesn't use try-except either so I think it's fine

File: tests/test_extract_lyrics.py (hunk 1 above)
Comment (MacyChan) [reply tests/test_extract_lyrics.py:26]: I have try-except on this function?
please clarify on the meeting, thanks!

File: src/pylyrics/extract_lyrics.py [hunk 3]
```diff
@@ -2,61 +2,11 @@
...
 
 import requests
-import os, sys
-import pandas as pd
-import json
-import urllib.parse
+from bs4 import BeautifulSoup
 import re
-import lyricsgenius
-from alive_progress import alive_bar
```
Comment (artanzand) [inline src/pylyrics/extract_lyrics.py:11]: Have you added the relevant packages to .toml?
If not, I will be doing this and pushing my toml because I need to be working with your functions

**Review by artanzand — COMMENTED** [review]
Resolution on adding the extra packages to toml

**Review by artanzand — APPROVED** [review]
Well done! Accepting changes to allow for other functions to run.

**Review by artanzand — APPROVED** [review]
Well done! Approving to allow the remaining dependancies to work
